addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js

```diff
@@ -205,8 +258,11 @@ export class ImageCrop extends Component {
         });
         delete this.media.dataset.resizeWidth;
         this.initialSrc = await applyModifications(this.media, {forceModification: true, mimetype: this.mimetype});
+        if(this.aspectRatio === "0/0"){
+            cropped = false;
```

Comment:
I think this condition can be varified when calling `_save()` function.
`this._save(cropped !== '0/0`)`

Reply:
It is for existing code.
https://github.com/odoo/odoo/blob/17.0/addons/web_editor/static/src/js/wysiwyg/widgets/image_crop.js#L273-L298

in this method, for case  =  "apply", we are not using any pre condition before _save() so it wouldn't know before actually adding "o_we_cropped" class that it is really needed or not.
so, it is for solving one extra bug that exist currently. it is nothing to do with this IMP.

Reply:
i have made separate commit for this purpose. 
Thank you!

Reply:
Thanks!! But don't you think this condition should be somewhere above  `cropped` is used ?
as well as add small comment, for Why?